store/blocks/desktop.json

```diff
@@ -33,14 +33,23 @@
   "flex-layout.row#sub-container.desktop": {
     "children": [
       "flex-layout.col#product-description",
-      "flex-layout.col#quantity.desktop",
+      "flex-layout.col#quantity-pricing.desktop",
       "flex-layout.row#price-remove"
     ],
     "props": {
       "preserveLayoutOnMobile": "true",
       "colSizing": "auto"
     }
   },
+  "flex-layout.col#quantity-pricing.desktop": {
+    "children": [
+      "flex-layout.col#quantity.desktop",
+      "flex-layout.col#manual-pricing"
```

Comment:
couldn't this go in the `flex-layout.col#quantity.desktop` block instead of creating another block for it?

Reply:
Sure!